panels/site_sections/panel_app_management.py

```diff
@@ -231,3 +231,26 @@ def everything(self, out, session):
                 ' / '.join(app.tech_needs_labels) + (' / ' if app.other_tech_needs else '') + app.other_tech_needs,
                 app.applied.strftime('%Y-%m-%d')
             ] + panelists)
+
+    def staff_schedule(self, session, attendee_id):
+        attendee = session.attendee(attendee_id)
+        events = defaultdict(lambda: defaultdict(lambda: (1, '')))
+        for pa in attendee.panel_applications:
```

Comment:
Using super-short variables in loops like this is a bit of an anti-pattern (something done commonly that is not best practice). It's much easier to read the code if you use a meaningful variable name instead of a super-short acronym. In this case you can actually use `app` which is both descriptive and short!